uint8 slave
uint8 index
uint16 value
---
string success